CleverPush/Source/CleverPushInstance.m

```diff
@@ -460,6 +460,8 @@ - (id)initWithLaunchOptions:(NSDictionary* _Nullable)launchOptions
     NSDictionary* userInfo = [launchOptions objectForKey:UIApplicationLaunchOptionsRemoteNotificationKey];
     if (userInfo) {
         startFromNotification = YES;
+    } else if (!launchOptions) {
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(onDidFinishLaunchingNotification:) name:UIApplicationDidFinishLaunchingNotification object:nil];
```

Comment:
This probably only works when the SDK is initialized early enough?
Many customers initialize our SDK delayed and not in the AppDelegate.

See also: https://stackoverflow.com/a/4361024/1418417